_projects/optimize_caching.md

```diff
@@ -1,25 +1,29 @@
 ---
 collaborating_projects:
   - coala
-desc: "Improve coala's performance by improving file caching and other performance bottlenecks."
+desc: "Improve coala's performance by improving file loading using factory
+      classes and other performance bottlenecks."
 difficulty: medium
 initiatives:
   - GSoC
   - RGSoC
 issues:
-  - "https://github.com/coala/coala/issues/2541"
-  - "https://github.com/coala/coala/issues/2699"
+  - "https://github.com/coala/coala/issues/3530"
   - "https://github.com/coala/coala/issues/2912"
+  - "https://github.com/coala/coala/issues/2541"
```

Comment:
why moved?

Reply:
Because I wanted to keep the sequence of issues in line with the timeline
The first is about `FileFactory` (implemented)
The second is  a general thread about Performance Optimization (one of them being implementing a new file caching module to replace linecache being used in the current implementation of `FileFactory`; to be implement in phase2 week1)
The third (which was moved) ignore directory will take the majority of the coding phase 2
The last one is cache control (phase3)

Reply:
Okay with me